Makefile

```diff
@@ -2,17 +2,21 @@
 SHELL = /bin/bash
 .SHELLFLAGS = -o pipefail -c
 
+# base path for Lexicon document tree (for lexgen)
+LEXDIR?=../atproto/lexicons
+
 .PHONY: help
 help: ## Print info about all commands
 	@echo "Commands:"
 	@echo
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "    \033[01;32m%-20s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: all
-all: build
-
 .PHONY: build
-build: ## Build all executables
+build: ## Build everything (to Go cache, not binaries)
```

Comment:
but why?

Reply:
`go build ./...` was running much faster for me on my laptop, and would find errors in all code, including any binaries not yet including the link of specific binaries to build. 

When I tried to time this just now I couldn't reproduce the different in build times though, it was only a second or two of different. So I can rip this part of the changes out if you think not worth it.

Reply:
yeah, i'm just really used to `make build` building everything and being able to run the binaries. Maybe we can add a `make compile` or `make check-compile` type thing to do what you're after? Its kinda misleading to have build not produce any artifacts